### ISSUE TITLE
Could not contact scylla.sh (500 Error)

### ISSUE THREAD

**Issue by user333222** [khast3x/h8mail#106]
*Issues not respecting the issue template will be closed without being read, thank you.*

### Checkbox

- [x] I have thoroughly checked the answer is not in the [wiki](https://github.com/khast3x/h8mail/wiki).
Yes
### Env

* h8mail version: Version 2.5.4 - "ROCKSMASSON.4"  
* Python version: Pyhton3.7
* Operating System: Kail Linux 64bit

### Description

Getting a 500 error for scylla.sh, not sure why. Ive changed my connection/VPNS and still get the same error. It use to work but now doesn't. Not too sure why. 

### What I Did 

(just random email, does this with any eamil entered) 

Command typed: sudo h8mail -t dhddh@yahoo.com  

Output: 
[~] Target factory started for dhddh@yahoo.com
[~] [dhddh@yahoo.com]>[hunter.io public]
[>] Found 0 related emails for dhddh@yahoo.com using hunter.io (public)
[~] [dhddh@yahoo.com]>[scylla.sh]
[!] Could not contact scylla.sh for dhddh@yahoo.com
500
<Response [500]>

```

**Comment by manu1907lama** [comment]
i have the same problem 
:(

**Comment by user333222** [comment]
I found this comment on there site about queries:  https://scylla.sh/api

You will get a 500 error if your query is incorrect. Use the fields listed above to guide you. Full query syntax (including wildcards) are supported.

Maybe they changed the way the API works?

**Comment by Zhindonm** [comment]
I changed the capital letters on ```get_scylla()``` to lower case

**Example**

Notice how ```Email``` & ```Password``` strings within the conditional clause have been changed.

Changed from  

```python
if user_query == "email":
                uri_scylla = 'Email: "' + self.target + '"'
elif user_query == "password":
                uri_scylla = 'Password: "' + self.target + '"'
```

*Changed to*

```python
if user_query == "email":
                uri_scylla = 'email: "' + self.target + '"'
elif user_query == "password":
                uri_scylla = 'password: "' + self.target + '"'
```

This is in the file h8mails/utils/classes.py

**Comment by Zhindonm** [comment]
@khast3x do you think this is the appropriate solution?

**Comment by user333222** [comment]
I did the same thing, but how do i get h8mail to use that new python file, when i ran it, it didn't use the updated python script

**Comment by user333222** [comment]
Got it, run it like this 
python3 -m h8mail -t test@mail.com

**Comment by user333222** [comment]
I can't seem to find a way to query wit ip. It seems it maybe of been disabled. not sure

**Comment by khast3x** [comment]
Hey all,

I have contacted the scylla API maintainer and we are currently reviewing this issue. @Zhindonm 's fix looks good, as well as @user333222 's fix for IP & username. 
Thank you all for your inputs as it really helps me out, feel free to update this issue with additional comments until we get this fixed.
Also sorry about `classes.py` I really have to refactor it one day.

Cheers!

**Comment by acaceres2176** [comment]
Hey folks, maintainer of scylla.sh here! I'm making fixes on my end, we accidentally changed some small things in the API and broke a bunch of people's tools (including this h8mail module). We're working to fix it ASAP so standby!

**Comment by acaceres2176** [comment]
Hey folks, found the issue. @khast3x this should now work. The issue was that the API changed a while back to have the keys be lowercase in the Lucene query syntax. In order to not break people's tools (ha), I made it such that the tool accepted lowercase and just lower'd it for users. The same wasn't done when the backend was refactored, so I went ahead and implemented it. Sorry for the interrupt in service!
